hs-abci-test-utils/src/Tendermint/Utils/Events.hs

```diff
@@ -0,0 +1,33 @@
+module Tendermint.Utils.Events where
```

Comment:
This module gives off different vibes than the rest in here. This one is actually used in application code (i.e., `nameservice/types`). The rest are only used in tests.